### PR TITLE
Nuxt module updates

### PR DESCRIPTION
**Pull request by andrew-boyd**

### CODE REVIEW

File: packages/vue/src/composables/defineFormKitConfig.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import { DefaultConfigOptions } from '../index'
+
+export function defineFormKitConfig(
+  config: DefaultConfigOptions = {}
+): () => DefaultConfigOptions {
+  return () => config
+}
```
Comment (andrew-boyd) [inline packages/vue/src/composables/defineFormKitConfig.ts:7]: New `defineFormKitConfig` function to provide typings

File: packages/nuxt/src/module.ts [hunk 2]
```diff
@@ -79,12 +78,6 @@ const module: NuxtModule<ModuleOptions> = defineNuxtModule<ModuleOptions>({
 
         ${configPathExists ? `import importedConfig from '${configPath}'` : ''}
```
Comment (andrew-boyd) [inline packages/nuxt/src/module.ts:79]: Import statement for the imported config that will end up in the module

File: packages/nuxt/playground/formkit.config.ts [hunk 3]
```diff
@@ -1,61 +1,8 @@
...
-
-export default config
+import { defineFormKitConfig } from '@formkit/vue'
+
+export default defineFormKitConfig({
+  theme: 'genesis',
+  // expecation is that useRuntimeConfig will be available
+  // when our defineFormKitConfig return function is called
+  test: useRuntimeConfig().test,
+})
```
Comment (andrew-boyd) [inline packages/nuxt/playground/formkit.config.ts:8]: Accessing `useRuntimeConfig()` inside of our imported `formkit.config.ts` results in 500 error `nuxt instance unavailable`

File: packages/nuxt/src/module.ts
Comment (justin-schroeder) [inline packages/nuxt/src/module.ts:1]: I think we need to remove `write: true` from this file.

**Review by justin-schroeder — CHANGES_REQUESTED** [review]

**Pull request merged**